src/index.js

```diff
@@ -90,12 +90,40 @@ class BodyBuilder {
    * @returns {BodyBuilder} Builder class.
    */
   sort(field, direction = 'asc') {
-    this._body.sort = {
-      [field]: {
-        order: direction
+    this._body.sort = this._body.sort || [];
+    var self = this;
+
+    if (_.isArray(field)) {
+
+        if(_.isPlainObject(this._body.sort)) {
+            this._body.sort = [this._body.sort];
+        }
+
+        if(_.isArray(self._body.sort)) {
+            _.each(field, function (sorts) {
+                _.each(sorts, function(value, key) {
+                    sortMerge(self._body.sort, key, value);
```

Comment:
Would this still work if I did 
```
.sort([{
  "channel": {
    "order": "desc"
  }
}])
```
We probably need to support this syntax as well.